test/functional/site/server.js

```diff
@@ -50,6 +53,31 @@ Server.prototype._setupRoutes = function () {
                 res.sendStatus(404);
             });
     });
+
+    this.app.post('/file-upload', upload.any(), function (req, res) {
+        // NOTE: uploadedFileArrays is an array which includes the file arrays which
+        // were added with the browsers.
+        // uploadedFileArrays = [browserXUploadedFiles, browserYUploadedFiles, ...];

```

Comment:
Well, it may well be enough, but since we decided that it's a rough place and we should leave a comment here, I would go with a couple of words in addition.
```
        // NOTE: uploadedFileArrays contains arrays of files
        // added for upload from each of the browsers.
        // uploadedFileArrays = [browserXUploadedFiles, browserYUploadedFiles, ...];
```